src/tests/performance/connections.rs

```diff
@@ -141,14 +141,15 @@ async fn p002_connections_load() {
         let node_addr = node.addr();
 
         let mut synth_sockets = Vec::with_capacity(synth_count);
-        let mut ips = IPS.to_vec();
+        // let mut ips = IPS.to_vec();
```

Comment:
let's remove this line in some future PR

Reply:
done